cli/command/telemetry_utils.go

```diff
@@ -100,6 +101,63 @@ func startCobraCommandTimer(mp metric.MeterProvider, attrs []attribute.KeyValue)
 	}
 }
 
+// basePluginCommandAttributes returns a slice of attribute.KeyValue to attach to metrics/traces
+func basePluginCommandAttributes(plugincmd *exec.Cmd, streams Streams) []attribute.KeyValue {
+	pluginPath := strings.Split(plugincmd.Path, "-")
+	pluginName := pluginPath[len(pluginPath)-1]
```

Comment:
Only `docker-[a-z][a-z0-9]*` binaries are considered a valid plugins anyway, so I think it's fine to simplify it to just a single `Cut` (to avoid allocating memory).

```suggestion
	_, pluginName, _  := strings.Cut(pluginCmd.Path, "-")
```

Reply:
We need to split after the last `-`, `strings.Cut` splits after the first occurrence and we have `cli-plugins` in the plugins' path

Reply:
Oh right, I missed that's a full path 🙈